sdk/keyvault/azkeys/README.md

```diff
@@ -220,14 +237,14 @@ func ExampleClient_UpdateKeyProperties() {
 		Tags: map[string]string{
 			"Tag1": "val1",
 		},
-		KeyAttributes: &azkeys.KeyAttributes{
-			RecoveryLevel: azkeys.CustomizedRecoverablePurgeable.ToPtr(),
+		Properties: &azkeys.Properties{
+			RecoveryLevel: azkeys.DeletionRecoveryLevelCustomizedRecoverablePurgeable.ToPtr(),
```

Comment:
Wow, this is really long. Is this change to create a standard for enums to have the enum type prefix the name?